docs/source/prodflow.rst

```diff
@@ -23,16 +24,41 @@ Data
 ------------
 Data collected in NEXT detectors is in form of *Raw Waveforms* (**RWF**, in ADCs). The following production flow is used over the **RWFs** for both type of Triggers:
 
-  .. image:: images/prodflow_data.png
+  .. image:: images/prodflow_data2.png
     :width: 850
 
  * Sensor parameters for :doc:`irene` need to be updated before any reconstruction. These are obtained from calibration using :doc:`phyllis`, :doc:`trude`, and :doc:`berenice` IC cities. These parameters are updated regularly during detector operation, and are stored in a database. Updates to the database are pushed to the repository, so keeping up to date with the main branch is very much recommended.
  * Correction maps are obtained from Krypton events (Trigger 1) using :doc:`ICAROS`. Official production correction maps can be found in :doc:`production`. A complete review about how this maps are produced can be found in "How to :ref:`krmaps`".
- * PSFs are nedeed to run Richardson Lucy deconvolution (:doc:`beersheba`). Krypton events (Trigger 1) are used under a specific configuration of :doc:`penthesilea` and :doc:`eutropia`. Official production PSFs for deconvolution can be found in :doc:`production`. A review about how this PSFs are produced can be found in "How to :ref:`psfdeco`".
+ * (*) Krypton events (Trigger 1) are used under a specific configuration of :doc:`sophronia` and :doc:`eutropia`. Official production PSFs for deconvolution can be found in :doc:`production`.
+ * PSFs are nedeed to run Richardson Lucy deconvolution (:doc:`beersheba`).  A review about how this PSFs are produced can be found in "How to :ref:`psfdeco`".
+ * (**) :doc:`sophronia` and :doc:`esmeralda` / :doc:`beersheba` receive different correction maps. During production, :doc:`sophronia` uses a preliminary correction map from the previous day to generate corrected hits, aiding in real-time data monitoring to catch potential issues early on. Meanwhile, :doc:`esmeralda` and :doc:`beersheba` apply the final corrections using a map created from the data of the ongoing Run. As this Run's processing concludes by the end of the day, the corresponding correction maps are only accessible the following day. For the **MC** process, only one correction is needed, and there is no distinction between applying it in :doc:`sophronia` or in the :doc:`esmeralda` / :doc:`beersheba` phase.
```

Comment:
Can we highlight the words "previous" and "ongoing"?

Reply:
done